packages/ToggleSwitch/ToggleSwitch.scss

```diff
@@ -0,0 +1,51 @@
+@import '~@tds/core-colours/colours';
+
+.switch {
+  position: relative;
+  cursor: pointer;
+  display: inline-block;
+  width: 44px;
+  min-width: 44px;
+  height: 22px;
+  border-radius: 22px;
+  background-color: $color-gainsboro;
+}
+
+.slider {
+  position: absolute;
+  width: 18px;
+  height: 18px;
+  top: 2px;
+  left: 2px;
+  border-radius: 18px;
+  background-color: white;
+  transition: left 0.25s;
+}
+
+.switchOn {
```

Comment:
This is a nitpik: if you were to prepend the key-value pair `composes: switch;`, then CSS Modules would automatically add the `.switch` class in your React component; and all you would need to add is `styles.switchOn` to your JSX, which adds both the `switchOn` and `switch` classes in the final component.

Your method works fine as well, so this change isn't absolutely necessary.

Reply:
Hmm, I'm sure that you're suggesting this for a good reason, but I'm not totally sure I understand the suggestion here.

Since this sounds like a _"nice to have"_, maybe we can do a patch release later the improve the styles?